example/Index.re

```diff
@@ -1,11 +1,24 @@
 open Express;
 
-let __dirname: option string = [%bs.node __dirname];
+let app = express ();
 
-let app = Express.express ();
+App.useOnPath app path::"/" @@ Middleware.fromF (fun _ _ next => {
+  Js.log "Request received"; 
+     /* This will be printed for every request */
+  next Js.undefined [@bs]
```

Comment:
Ok, so both of these seem like premature optimization and putting implementations concerns over interface concerns, which I think should at least have proper justification. And this should still be possible to do even if we provide and promote a more convenient alternative, right?